addons/mrp/tests/test_procurement.py

```diff
@@ -183,17 +183,20 @@ def test_date_propagation(self):
         self.assertEqual(mo_form.product_qty, 10, 'the quantity to produce is not good relative to the move')
 
         mo = mo_form.save()
-
         # Confirming mo create finished move
         move_orig = self.env['stock.move'].search([
             ('move_dest_ids', 'in', move_dest.ids)
         ], limit=1)
+        move_orig.write({'propagate_date': True, 'propagate_date_minimum_delta': 1})
```

Comment:
the manufacture rule already got propagate date true en delta 1 isn't it? why this change?